tensorboard/compat/proto/plugin_text.proto

```diff
@@ -0,0 +1,27 @@
+/* Copyright 2017 The TensorFlow Authors. All Rights Reserved.
```

Comment:
Same for this

https://github.com/tensorflow/tensorboard/blob/master/tensorboard/plugins/text/plugin_data.proto

Reply:
replaces plugin_text.proto and plugin_pr_curve.proto with plugin_data.proto will lead to name confliction. How about using soft links?